todo_app/data/item.py

```diff
@@ -5,7 +5,15 @@ def __init__(self, id, name, status = Status.ToDo.value):
         self.id = id
         self.name = name
         self.status = status
+
+    def __init__(self, item):
+        self.id = item["_id"]
+        self.name = item["name"]
+        self.status = item["status"]
```

Comment:
Given this relies on a specific style of item, I'd be tempted to solve it with a classmethod like the one below but recognising that it's from a Mongo item